cluster.tf

```diff
@@ -12,7 +12,10 @@ resource "aws_eks_cluster" "this" {
   enabled_cluster_log_types = var.cluster_enabled_log_types
   role_arn                  = local.cluster_iam_role_arn
   version                   = var.cluster_version
-  tags                      = var.tags
+  tags                      = merge(
+    var.cluster_tags,
```

Comment:
I think better will be using:
```
tags = merge(
        var.tags,
        var.cluster_tags)
```

as then you will be able to overwrite some common tags by cluster tags.

As this is how [merge](https://www.terraform.io/docs/language/functions/merge.html) is working

Reply:
thanks for the feedback, and pointing out how merge's order of precedence works.  I'll make the update and ping you when I'm done.